Twig/PagerfantaExtension.php

```diff
@@ -103,6 +103,7 @@ private function createRouteGenerator($options = array())
                 'routeName'     => null,
                 'routeParams'   => array(),
                 'pageParameter' => '[page]',
+                'omitFirstPage' => true
```

Comment:
The default value should be `false` to actually revert the BC break introduced by #131.